### PR TITLE
Influx password in configuration file

### PR DESCRIPTION
**Pull request by Polina-Gubina**
Implement part of #33 (for influx password)

AWX password still to be implemented

### CODE REVIEW

File: delatore/configuration.py [hunk 1]
```diff
@@ -25,12 +26,14 @@ def read_config(config_file=RESOURCES['config.ini']):
     config.read(config_file)
     defaults = config.defaults()
     if defaults:
-        token = defaults.get('token', os.getenv('token'))
-        chat_id = defaults.get('chat_id', os.getenv('chat_id'))
-        return BotConfig(token, chat_id)
+        token = defaults.get('token')
```
Comment (outcatcher) [inline delatore/configuration.py]: Why removed using env var as value if configuration entry is missing?

File: delatore/sources/influx.py [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
 from influxdb import InfluxDBClient
 
+from delatore.configuration import BOT_CONFIG
```
Comment (outcatcher) [inline delatore/sources/influx.py]: Using absolute import

File: tests/test_config.py [hunk 3]
```diff
@@ -1,10 +1,12 @@
+import pytest
 from delatore.configuration import BOT_CONFIG, BotConfig, read_config
```
Comment (anton-sidelnikov) [inline tests/test_config.py]: make import relative

File: delatore/sources/influx.py [hunk 4]
```diff
@@ -8,6 +8,7 @@
 
 from influxdb import InfluxDBClient
 
+import delatore.configuration.BOT_CONFIG as BC
```
Comment (anton-sidelnikov) [inline delatore/sources/influx.py]: make import relative

**Comment by outcatcher** [comment]
Outdated

**Pull request closed without merging**